sdk/src/main/kotlin/com/processout/sdk/api/model/response/POCardIssuerInformationResponse.kt

```diff
@@ -0,0 +1,23 @@
+package com.processout.sdk.api.model.response
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+internal data class POCardIssuerInformationResponse(
```

Comment:
PO prefix could be dropped since class is not public.

Reply:
Yes, it applies to all request/response models, need to rename them all at once, so far keeping consistency.